app/views/reservations/current_reservations.html.erb

```diff
@@ -33,8 +33,10 @@
 
 <hr>
 <%= link_to "Print this page", "#", onclick: "print();", class: 'btn btn-default' %>
-<%# if (@reservation.status == "checked out") or (@reservation.status == "overdue") %>
-  <%#= link_to "Email checkout receipt", {:controller => :reservations, :action => "checkout_email", :id => @reservation}, :method => "get", :class => 'btn btn-default' %>
-<%# elsif @reservation.status.include?("returned") %>
-  <%#= link_to "Email return receipt", {:controller => :reservations, :action => "checkin_email", :id => @reservation}, :method => "get", :class => 'btn btn-default' %>
-<%# end %>
+<% if @reservation %>
+  <% if (@reservation.status == "checked out") or (@reservation.status == "overdue") %>
+    <%= link_to "Email checkout receipt", {:controller => :reservations, :action => "send_receipt", :id => @reservation}, :method => "get", :class => 'btn btn-default' %>
```

Comment:
Can we update this hash syntax? Honestly, this almost feels like we just want a helper method for the e-mail receipt button, but that's beyond scope so don't worry about it!